js/selectivizr-min.js

```diff
@@ -0,0 +1,5 @@
e,b=x(d,b,c);if(b!=d)a.className=b,a.parentNode.className+=j}function x(a,b,c){var d=RegExp("(^|\\s)"+b+"(\\s|$)"),e=d.test(a);return c?e?a:a+p+b:e?a.replace(d,l).replace(y,l):a}function k(a,b,c){a.attachEvent("on"+b,c)}function t(a,b,c){b||(b=U);a.substring(0,2)=="//"&&(a=b.substring(0,b.indexOf("//"))+a);if(/^https?:\/\//i.test(a))return!c&&b.substring(0,b.indexOf("/",8))!=a.substring(0,a.indexOf("/",8))?null:a;if(a.charAt(0)=="/")return b.substring(0,b.indexOf("/",8))+a;b=b.split(/[?#]/)[0];a.charAt(0)!="?"&&b.charAt(b.length-1)!="/"&&(b=b.substring(0,b.lastIndexOf("/")+1));return b+a}function z(a){if(a)return o.open("GET",a,!1),o.send(),(o.status==200?o.responseText:j).replace(V,j).replace(W,function(b,c,d,e,f,g){b=z(t(d||f,a));return g?"@media "+g+" {"+b+"}":b}).replace(X,function(b,c,d,e){d=d||j;return c?b:" url("+d+t(e,a,!0)+d+") "});return j}function Y(){N();s.length>0&&setInterval(function(){for(var a=0,b=s.length;a<b;a++){var c=s[a];if(c.disabled!==c.a)c.disabled?(c.disabled=!1,c.a=!0,c.disabled=!0):c.a=c.disabled}},250)}if(!/*@cc_on!@*/true){var f=document,r=f.documentElement,o=function(){if(i.XMLHttpRequest)return new XMLHttpRequest;try{return new ActiveXObject("Microsoft.XMLHTTP")}catch(a){return null}}(),m=/MSIE (\d+)/.exec(navigator.userAgent)[1];if(!(f.compatMode!="CSS1Compat"||m<6||m>8||!o)){var A={NW:"*.Dom.select",MooTools:"$$",DOMAssistant:"*.$",Prototype:"$$",YAHOO:"*.util.Selector.query",Sizzle:"*",jQuery:"*",dojo:"*.query"},w,s=[],q=[],R=0,Q=!0,P="slvzr",V=/(\/\*[^*]*\*+([^\/][^*]*\*+)*\/)\s*?/g,W=/@import\s*(?:(?:(?:url\(\s*(['"]?)(.*)\1)\s*\))|(?:(['"])(.*)\3))\s*([^;]*);/g,X=/(behavior\s*?:\s*)?\burl\(\s*(["']?)(?!data:)([^"')]+)\2\s*\)/g,M=/^:(empty|(first|last|only|nth(-last)?)-(child|of-type))$/,D=/:(:first-(?:line|letter))/g,E=/((?:^|(?:\s*})+)(?:\s*@media[^{]+{)?)\s*([^\{]*?[\[:][^{]+)/g,I=/([ +~>])|(:[a-z-]+(?:\(.*?\)+)?)|(\[.*?\])/g,O=/(:not\()?:(hover|enabled|disabled|focus|checked|target|active|visited|first-line|first-letter)\)?/g,S=/[^\w-]/g,L=/^(INPUT|SELECT|TEXTAREA|BUTTON)$/,K=/^(checkbox|radio)$/,u=m>6?/[\$\^*]=(['"])\1/:null,G=/([(\[+~])\s+/g,H=/\s+([)\]+~])/g,T=/\s+/g,y=/^\s*((?:[\S\s]*\S)?)\s*$/,j="",p=" ",l="$1",B=f.getElementsByTagName("BASE"),U=B.length>0?B[0].href:f.location.href;(function(){for(var a,b,c=0;c<f.styleSheets.length;c++)if(b=f.styleSheets[c],b.href!=j&&(a=t(b.href)))b.cssText=b.rawCssText=C(z(a))})();(function(a,b){function c(){try{r.doScroll("left")}catch(a){setTimeout(c,50);return}d("poll")}function d(c){if(!(c.type=="readystatechange"&&f.readyState!="complete")&&((c.type=="load"?a:f).detachEvent("on"+c.type,d,!1),!e&&(e=!0)))b.call(a,c.type||c)}var e=!1,h=!0;if(f.readyState=="complete")b.call(a,j);else{if(f.createEventObject&&r.doScroll){try{h=!a.frameElement}catch(g){}h&&c()}k(f,"readystatechange",d);k(a,"load",d)}})(i,function(){for(var a in A){var b,c,d=i;if(i[a]){for(b=A[a].replace("*",a).split(".");(c=b.shift())&&(d=d[c]););if(typeof d=="function"){w=d;Y();break}}}})}}})(this);
```

Comment:
Line is too long.<br>'b' is already defined.<br>'d' is already defined.<br>Missing 'new' prefix when invoking a constructor.<br>Expected '{' and instead saw 'return'.<br>Expected a conditional expression and instead saw an assignment.<br>Missing semicolon.<br>Don't make functions within a loop.<br>Expected an assignment or function call and instead saw an expression.<br>Expected '{' and instead saw 'switch'.<br>Expected a 'break' statement before 'case'.<br>Expected '{' and instead saw 'b'.<br>Confusing use of '!'.<br>Expected '{' and instead saw 'f'.<br>Too many errors. (100% scanned).